test/aqua/test_grover.py

```diff
@@ -145,8 +145,12 @@ def test_state_preparation_type_error(self):
         init_state = Zero(2)
         oracle = QuantumCircuit(2)
         oracle.cz(0, 1)
-        with self.assertRaises(TypeError):
-            Grover(oracle=oracle, state_preparation=init_state)
+        try:
+            warnings.filterwarnings(action="ignore", category=DeprecationWarning)
```

Comment:
Could you add a comment here stating what is being suppressed - I assume this is some underlying library that we cannot directly address. Otherwise tests on Grover should be emitting warnings unless it is specifically testing the now deprecated quantities following the recent refactor.